### ISSUE TITLE
Question: Flipper for windows desktop

### ISSUE THREAD

**Issue by chaitanyaa-simform** [facebook/flipper#5674]
I am installing flipper for windows desktop but its gets downloading .tgz file which is not supported for windows. Is flipper support windows now??

**Comment by masztalski** [comment]
Yes, I have solved it with NPM . On command line type npx flipper-server . Then it will run in browser tab. You can install it as PWA. However as PWA it works for me only if I have started server manually each time by command line :)
